src/components/Li.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <li v-if="this.$slots.default" class="slot">
+    <slot />
+  </li>
+  <li v-else :class="{ disabled: !options.link.link && !options.link.onClick, options: true }">
```

Comment:
Hm, here you assume knowledge about the link object (`LinksOptions`).
At least, I would prefer a method returning whether it is disabled or not.

Reply:
Resolved that though link as an optinal parameter in Li.